distributed/client.py

```diff
@@ -23,6 +23,8 @@
 import warnings
 import weakref
 
+from bokeh.resources import INLINE
```

Comment:
`bokeh` is an optional dependency, so we tend to do all `bokeh`-related imports inside functions and methods. This allows us to raise `ImportError`s when users call a function that requires `bokeh`, e.g. `get_task_stream`, but not when doing other non-`bokeh` operations like `import distributed`